Memory Stream.txt

```diff
@@ -0,0 +1,147 @@
+OBJECT Codeunit 50777 MemoryStream
+{
+  OBJECT-PROPERTIES
+  {
+    Date=20-09-17;
+    Time=[ 7:32:02 PM];
+    Modified=Yes;
+    Version List=;
+  }
+  PROPERTIES
+  {
+    OnRun=BEGIN
+            UnitTest();
+          END;
+
+  }
+  CODE
+  {
+    VAR
+      MemoryStream@1160530000 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.MemoryStream";
+      StreamWriter@1160530001 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.StreamWriter";
+      StreamReader@1160530002 : DotNet "'mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089'.System.IO.StreamReader";
+
+    PROCEDURE Create@1160530001(InitialSize@1160530000 : Integer);
+    BEGIN
+      MemoryStream := MemoryStream.MemoryStream(InitialSize);
```

Comment:
Practical would be the implementation of the empty constructor too.

Reply:
Besides, reading and writing would be interesting and I think dispose is very important.